adapters/avocet/avocet_test.go

```diff
@@ -0,0 +1,291 @@
+package avocet
+
+import (
+	"encoding/json"
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "avocet", NewAvocetAdapter("https://bid.staging.avct.cloud/ortb/bid/5e722ee9bd6df11d063a8013"))
+}
+
+func TestAvocetAdapter_MakeRequests(t *testing.T) {
+	type fields struct {
+		Endpoint string
+	}
+	type args struct {
+		request *openrtb.BidRequest
+		reqInfo *adapters.ExtraRequestInfo
+	}
+	type reqData []*adapters.RequestData
+	tests := []struct {
+		name     string
+		fields   fields
+		args     args
+		want     []*adapters.RequestData
+		wantErrs []error
+	}{
+		{
+			"makes POST request with JSON content",
+			fields{Endpoint: "https://bid.avct.cloud"},
+			args{
+				&openrtb.BidRequest{},
+				nil,
+			},
+			reqData{
+				&adapters.RequestData{
+					Method: http.MethodPost,
+					Uri:    "https://bid.avct.cloud",
+					Body:   []byte(`{"id":"","imp":null}`),
+					Headers: map[string][]string{
+						"Accept":       {"application/json"},
+						"Content-Type": {"application/json;charset=utf-8"},
+					},
+				},
+			},
+			nil,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			a := &AvocetAdapter{
+				Endpoint: tt.fields.Endpoint,
+			}
+			got, got1 := a.MakeRequests(tt.args.request, tt.args.reqInfo)
+			if len(got) != len(tt.want) {
+				t.Errorf("AvocetAdapter.MakeRequests() got %v requests, wanted %v requests", len(got), len(tt.want))
+			}
+			if len(got) == len(tt.want) {
+				for i := range tt.want {
+					if !reflect.DeepEqual(got[i], tt.want[i]) {
+						t.Errorf("AvocetAdapter.MakeRequests() got = %v, want %v", got[i], tt.want[i])
+					}
+				}
+			}
+			if !reflect.DeepEqual(got1, tt.wantErrs) {
+				t.Errorf("AvocetAdapter.MakeRequests() got1 = %v, want %v", got1, tt.wantErrs)
+			}
+		})
+	}
+}
+
+func TestAvocetAdapter_MakeBids(t *testing.T) {
+	type fields struct {
+		Endpoint string
+	}
+	type args struct {
+		internalRequest *openrtb.BidRequest
+		externalRequest *adapters.RequestData
+		response        *adapters.ResponseData
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		want   *adapters.BidderResponse
+		errs   []error
+	}{
+		{
+			"204 No Content indicates no bids",
```

Comment:
Nice use of table driven tests. Using this format, please consider specifying the struct field name to improve readability. Ie:

```
{
	name: ...,
	fields: ...,
	args: ...,
	want: ...,
	errs: ...,
}
```